File: CodeForces/101-200/118/A/118A.ts
// Problem 118A: String Task
// https://codeforces.com/contest/118/problem/A
// Compiled with: npx 9.2.0, tsc Version 5.9.3
// Executed with: node v20.19.4
// Linux version: Linux kernel 6.17.0-5-generic
// Written on: October 26th, 2025
// Codeforces language used: Cannot be submitted

import * as readline from 'node:readline/promises';
import { stdin as input, stdout as output } from 'node:process';

async function main() {
    const rl = readline.createInterface({ input, output });

    let input_string: string = await rl.question('');
    rl.close();

    input_string = input_string.toLowerCase();
    let answer: string = "";
    const vowels = "aeiouy";

    for (let i = 0; i < input_string.length; i++) {
        if (vowels.indexOf(input_string[i]) == -1) {
            answer += ".";
            answer += input_string[i];
        }
    }
    console.log(answer)
    
}

main();

File: CodeForces/101-200/118/A/118A.java
// Problem 118A: String Task
// https://codeforces.com/contest/118/problem/A
// Compiled with: openjdk 25 2025-09-16
// Executed with: javac 25
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 25th, 2025
// Codeforces language used: Java 21 64bit

import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input = sc.nextLine().toLowerCase();
        String vowels = "aeiouy";
        for (int i = 0; i < input.length(); i++) {
            if (vowels.indexOf(input.charAt(i)) == -1){
                System.out.print("." + input.charAt(i));
            }
        }
        System.out.print("\n");
        sc.close();
    }
}
